### PR TITLE
update sfrp sdk to 2023-11-01 

### PR DESCRIPTION
**Pull request by mwesigwaguma**
## Mandatory Checklist

- Please choose the target release of Azure PowerShell
  - [x] [General release](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Public preview](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Private preview](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Engineering build](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] N/A

- [ ] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For SDK-based development mode, update `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`.
        * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. 
    * For autorest-based development mode, include the changelog in the PR description.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

File: src/ServiceFabric/ServiceFabric.Management.Sdk/README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ payload-flattening-threshold: 2
 use-extension:
```
Comment (msJinLei) [inline src/ServiceFabric/ServiceFabric.Management.Sdk/README.md:20]: READ.md is not in correct format. Please refer to https://github.com/Azure/azure-powershell/blob/main/src/KeyVault/KeyVault.Sdk/README.md to correct the file

**Review by msJinLei — CHANGES_REQUESTED** [review]

File: src/ServiceFabric/ServiceFabric.Management.Sdk/README.md (hunk 1 above)
Comment (mwesigwaguma) [reply src/ServiceFabric/ServiceFabric.Management.Sdk/README.md:21]: Made some changes but these files readme looks similar to the one mentioned above. Let me know if  I am missing something

**Comment by msJinLei** [comment]
```
[Exception] No input files provided.  Use --help to get help information or see https://aka.ms/autorest/cli for additional documentation error   

[Exception] No input files provided.  Use --help to get help information or see https://aka.ms/autorest/cli for additional documentation error 
```

**Comment by msJinLei** [comment]
@mwesigwaguma I upload the changes of generated sdks with your updated swagger commit. There are some compilation errors, which are caused by the breaking changes of management sdks. 
Please
- fix the compilation issues
  - you may adapt the codes from project ServiceFabricproject
  - don't change the codes from project ServiceFabric.Management.Sdk and ServiceFabricManagedClusters.Management.Sdk
- Let's check if there is a breaking change for cmdlet input and output if compilation is sucessful

**Pull request closed without merging**